### DISCUSSION TITLE
github_trending_2023-12-28/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme#670] [General]
# github_trending_2023-12-28/

微博热搜榜 排名 1.以中国式现代化全面推进强国建设 排名 2.大学生不要再发求捞表情包了 排名 3.新生儿遭月子中心月嫂虐待 排名 4.2023中国航天高燃时刻 排名 5.最后的绅士 排名 6.全面禁放烟花爆竹是时候终结了 排名 7.韩媒曝李善均遗书内容 排名 8.繁花里的papi酱 排名 9.马思纯闻香汗流浃背了吧

https://www.jobcher.com/github_trending_2023-12-28/

**Comment by jobcher** [comment]
沙发
